docs/reference/api-conventions.asciidoc

```diff
@@ -215,13 +215,16 @@ The supported units are:
 `m`:: minutes
 `s`:: seconds
 
-Some examples are:
+Assuming `now` is `2001-01-01 12:00:00`, some examples are:
 
-[horizontal]
-`now+1h`::              The current time plus one hour, with ms resolution.
-`now+1h+1m`::           The current time plus one hour plus one minute, with ms resolution.
-`now+1h/d`::            The current time plus one hour, rounded down to the nearest day.
-`2015-01-01||+1M/d`::   `2015-01-01` plus one month, rounded down to the nearest day.
+[options="header"]
```

Comment:
Please keep the definition list format instead of a table. Tables don't render terribly well in Docbook.